ubuntu.json

```diff
@@ -270,6 +270,11 @@
     "key_url": "https://www.rabbitmq.com/rabbitmq-signing-key-public.asc"
   },
   {
+    "alias" : "team-gcc-arm-embedded",
```

Comment:
Probably right - removed, thanks. I forgot about this one as i figured they wont be adding new sources anyway for the time being